pvHelpers/win_helpers.py

```diff
@@ -596,76 +596,71 @@ def __del__(self):
         super(WinPopen, self).__del__()
 
 
+def list_active_sessions():
+    return filter(
+        lambda s: s["State"] == win32ts.WTSActive and s["SessionId"] != 0, 

```

Comment:
is there a Microsoft doc for this? `s["SessionId"] != 0` means?

Reply:
sorry, will leave comment